assets/collections/comments.js

```diff
@@ -0,0 +1,2 @@
+var comments = new CommentList();
+var app = new AppView({collection: comments});
```

Comment:
These lines starting up the app shouldn't be in a `collections` folder. Best practice would put this in an `app.js` or `index.js` file to know what starts the actual app running.